### ISSUE TITLE
Bug: ERROR VPN settings: provider settings: server selection: the region specified is not valid: value is not one of the possible choices

### ISSUE THREAD

**Issue by Shinops** [qdm12/gluetun#1665]
### Is this urgent?

No

### Host OS

Ubuntu

### CPU arch

x86_64

### VPN service provider

NordVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2020-03-13T01:30:06Z (commit d0f678c)

### What's the problem 🤔

Server regions specified is not valid after latest update.

### Share your logs

```plain text
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================
Running version latest built on 2023-06-08T20:07:09.067Z (commit 4675572)
🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
route.table 254
2023-06-09T00:18:48+02:00 INFO [routing] default route found: interface eth0, gateway 192.168.89.1, assigned IP 192.168.89.5 and family v4
route.table 254
route.table 255
route.table 255
route.table 255
route.table 255
route.table 255
2023-06-09T00:18:48+02:00 INFO [routing] local ethernet link found: eth0
2023-06-09T00:18:48+02:00 INFO [routing] local ipnet found: 192.168.89.0/24
2023-06-09T00:18:48+02:00 INFO [firewall] enabling...
2023-06-09T00:18:49+02:00 INFO [firewall] enabled successfully
2023-06-09T00:18:49+02:00 INFO [storage] merging by most recent 17678 hardcoded servers and 17678 servers read from /gluetun/servers.json
2023-06-09T00:18:49+02:00 ERROR VPN settings: provider settings: server selection: for VPN service provider nordvpn: the region specified is not valid: value is not one of the possible choices: value poland, choices available are Africa, the Middle East and India, Asia Pacific, Europe, The Americas
```

### Share your configuration

```yml
gluetun:
    image: qmcgaw/gluetun
    container_name: gluetun
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - 8888:8888/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks
      - 8090:8090 # port for qbittorrent
    volumes:
      - $DOCKERDIR/appdata/gluetun:/gluetun
    environment:
      - VPN_SERVICE_PROVIDER=nordvpn
      - OPENVPN_USER=$USERNAMEVPN
      - OPENVPN_PASSWORD=$PASSWORDVPN
      - SERVER_REGIONS= "Poland"
      - TZ=$TZ
    network_mode: npm_proxy
    restart: always
```

**Comment by Shinops** [comment]
Similar to issue #1613, rolling back to v3.34.1 doesn't work also

**Comment by qdm12** [comment]
As the log mentions, the region is now the continent, you can use `SERVER_COUNTRIES` to specify the country.
I decided to bundle this and slightly break changes for once, since keeping both retro-compatible was a nightmare.

This was changed in https://github.com/qdm12/gluetun/commit/07459ee854f9609c1e9aa7d7161f9af04a59199b which affects the latest image, although I'm not 100% sure about why v3.34 complains about this.

Let's keep this opened so I update the wiki about it.

**Comment by qdm12** [comment]
Actually running:

```sh
docker run -it --rm --cap-add=NET_ADMIN -e VPN_SERVICE_PROVIDER=nordvpn -e OPENVPN_USER=a -e OPENVPN_PASSWORD=b -e SERVER_REGIONS=Poland --name gluetun qmcgaw/gluetun:v3.34.1
```

does work for me (same version `Running version v3.34.1 built on 2023-05-22T06:02:37.334Z (commit 6d48f9c)`), so I'm not sure what's going in your logs for v3.34.x images.

**Comment by qdm12** [comment]
Wiki got updated 👍

**Comment by Shinops** [comment]
Thank you, going back to the lastet version with Server_Countries as the parameter, it worked! Thank you also for the quick reply and editing on the wiki :)

**Comment by qdm12** [comment]
Fixed retro-compatibility in 4a0f9c36ba2aa5055f16f2444cc83c66521e7c48
